plenum/test/test_node.py

```diff
@@ -379,7 +379,7 @@ def __init__(self,
         self.pluginPaths = pluginPaths
 
         self.testNodeClass = testNodeClass
-        self.nodes = OrderedDict()  # type: Dict[str, TestNode]
+        self.nodes = SortedDict()  # type: Dict[str, TestNode]
```

Comment:
reverted.
But actually this is not obvious. It means we should either use `txnPoolNodeSet`, or choose not the last node to stop. We almost always stop the last node (assuming that it will not be next Primary, which is not true for `nodeSet` with `nodeNum=4`)